components/camel-pulsar/src/main/java/org/apache/camel/component/pulsar/PulsarProducer.java

```diff
@@ -20,11 +20,14 @@
 import org.apache.camel.Message;
 import org.apache.camel.NoTypeConversionAvailableException;
 import org.apache.camel.TypeConversionException;
+import org.apache.camel.component.pulsar.configuration.PulsarConfiguration;
 import org.apache.camel.component.pulsar.utils.message.PulsarMessageUtils;
 import org.apache.camel.support.DefaultProducer;
 import org.apache.pulsar.client.api.Producer;
 import org.apache.pulsar.client.api.ProducerBuilder;
 
+import java.util.concurrent.TimeUnit;
```

Comment:
Please run the sourcecheck profile. It will return warnings

Reply:
I ran the sourcecheck profile and it showed no errors.

Reply:
I reran and found the errors, but as they don't break the build and had scrolled off the screen I accepted the build completing successfully as a thumbs up. Will now fix.